lib/rails_cloud_tasks/rack/jobs.rb

```diff
@@ -15,6 +15,10 @@ def call(env)
           request = ::Rack::Request.new(env)
           job_args = extract_args(request)
 
+          RailsCloudTasks::Instrumentation.add_custom_attributes(
+            { arguments: job_args }
```

Comment:
That's a `request payload` so I'd recommend renaming to something related to... `request_body` for instance